app/javascript/vue/shared/content/attachments/multiple_open.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+
+  <!-- multiple options -->
+  <MenuDropdown
+    v-if="multipleOpenOptions.length > 1"
+    :listItems="multipleOpenOptions"
+    :btnClasses="'btn btn-light icon-btn thumbnail-action-btn'"
+    :position="'left'"
+    :btnIcon="'sn-icon sn-icon-open'"
+    :title="i18n.t('attachments.thumbnail.buttons.open')"
+    @menu-visibility-changed="handleMenuVisibilityChanged"
```

Comment:
Is this just supposed to be like an open/close or toggle event? Menu visibility sounds a bit strange